app/pages/resource/info/ResourceInfo.js

```diff
@@ -5,24 +5,47 @@ import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import Label from 'react-bootstrap/lib/Label';
 import Row from 'react-bootstrap/lib/Row';
 
+import FavoriteButton from 'shared/favorite-button';
 import ImageCarousel from 'shared/image-carousel';
 import WrappedText from 'shared/wrapped-text';
 
-function ResourceInfo({ resource, unit }) {
+ResourceInfo.propTypes = {
+  currentUser: PropTypes.object,
```

Comment:
- [ ] This could be renamed to `isLoggedIn` since that's the information the component needs to decide whether to render the favorite button or not.